client/src/components/Footer/Footer.css

```
@@ -0,0 +1,58 @@
+footer {
+    background-color:#B3B3B3;;
+    padding: 20px;
+}
+
+.footer-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.footer-headings {
+    font-size: 25px;
+    margin-bottom: 15px;
+    color: white;
+}
+
+.address,
+.contact-details,
+.services,
+.pages {
+    margin-bottom: 20px;
+}
+
+.address {
+    margin-bottom: 10px;
+}
+
+.no-list-style {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+a {
+    color: black;
+    text-decoration: none;
+}
+
+ul li {
+    margin-bottom: 5px;
+}
+
+/* Mobile-first design */
+@media (min-width: 768px) {
+
+    /* Desktop version */
+    .footer-container {
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
```

```suggestion
    /* Desktop version */
    footer {
        flex-direction: row;
        justify-content: space-evenly;
        gap: 20px; /* or whatever gap you want */
    }
```
we can use this on the footer if we apply the above logic.
the gap will change according to mobile/desktop
the justify-content prevents the need for the flex-basis percentages

and about the wrap, is the logic when the screen gets too small for 4 columns and you want it to snap over underneath? this is a good thing for us to think about re: breakpoints.

just thoughts:
maybe grid would be better then eg.
1 col x 4 row in mobile
2 col x 2 row in tablet
4 col x 1 row in desktop

---

I've committed the changes you've suggested.
The first point you made about flex-wrap is why I used it.